resources/js/rom.js

```diff
@@ -250,6 +250,11 @@ export default class ROM {
       let byte = 0x24;
       let file_byte = 0x05;
 
+      if (color_on === "random") {
+        const colorOptions = ["blue", "green", "yellow", "red"];
+        color_on = colorOptions[Math.floor(Math.random() * colorOptions.length)];
```

Comment:
This file has access to the `Prando` library, might be worth using that instead of `Math`

Reply:
I don't think we need to use Prando here.  I think it's fine for the color choice to be unseeded in this instance.

Reply:
Started to, but palette shuffle is `reset()`ing the instance in `this.rand`, and also Synack didn't agree to use it here, so I've left it as-is for the time being.